docs/physics/gammaray_deposition.ipynb

```diff
@@ -0,0 +1,573 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #3.    plt.plot(0,0)
</pre><p>labels and comments</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tardis-sn/tardis/pull/1698/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>